src/date-picker/date-picker.scss

```diff
@@ -62,58 +63,220 @@
   > div {
     white-space: nowrap;
     overflow: hidden;
-    width: $iui-xl;
+    width: $iui-date-picker-cell-width;
     text-align: center;
   }
 }
 
 @mixin iui-calendar-week {
   white-space: nowrap;
-  height: $iui-baseline * 3;
+  line-height: $iui-date-picker-cell-height;
   display: flex;
-  align-items: center;
-  justify-content: space-evenly;
   padding: 0 $iui-s;
+  margin-bottom: $iui-xxs;
 }
 
-@mixin iui-calendar-day {
+//#region Helper mixins
+@mixin iui-calendar-day-base {
   @include iui-focus;
   cursor: pointer;
-  display: inline-block;
-  width: $iui-baseline * 3;
-  line-height: $iui-baseline * 3;
+  text-align: center;
+  width: $iui-date-picker-cell-width;
+  height: $iui-date-picker-cell-height;
   border-radius: $iui-border-radius;
+  font-variant-numeric: tabular-nums;
+}
 
-  &.iui-today {
+@mixin iui-calendar-day-base-hover {
+  &:hover {
     font-weight: $iui-font-weight-semibold;
-    border-radius: 50%;
     @include themed {
-      background-color: rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-6));
+      color: t(iui-color-foreground-primary);
+      background-color: rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-6));
+    }
+  }
+}
+
+@mixin iui-calendar-day-today-base {
+  position: relative;
+  font-weight: $iui-font-weight-semibold;
+
+  &::before {
+    content: '';
+    position: absolute;
+    display: block;
+    width: $iui-date-picker-cell-height;
+    height: $iui-date-picker-cell-height;
```

Comment:
For me it doesn't look nice when circle is from the very top to the bottom. What if we reduce its size a bit? Here it is 32x32
![image](https://user-images.githubusercontent.com/36186912/159649401-81d7b99a-3100-4b25-8897-2409dd2783a7.png)

Reply:
Changed